test/common.bash

```diff
@@ -177,32 +199,12 @@ ch_tardir=$(readlink -ef "$CH_TEST_TARDIR")
 # shellcheck disable=SC2034
 ch_mounts="${ch_imgdir}/mounts"
 
-# Image information.
-# shellcheck disable=SC2034
-ch_tag=${CH_TEST_TAG:-NO_TAG_SET}  # set by Makefile; many tests don't need it
-# shellcheck disable=SC2034
-ch_img=${ch_imgdir}/${ch_tag}
-# shellcheck disable=SC2034
-ch_tar=${ch_tardir}/${ch_tag}.tar.gz
```

Comment:
It must work, but how do the run and build tests get `ch_img` and `ch_tar` now? Are the no longer necessary?